config/presets/fastnet.go

```diff
@@ -92,9 +92,12 @@ func fastnet() config.Config {
 	conf.Beacon.VotesLimit = 100
 
 	conf.POET.GracePeriod = 10 * time.Second
+	conf.POET.SubmitChallengeTimeout = 10 * time.Second
```

Comment:
Why is this not equal to `RequestTimeout`?

Reply:
removed SubmitChallengeTimeout timeout. By default we will submit challenges till end of cycle gap